src/Uno.AzureDevOps/Uno.AzureDevOps.Views/Content/ProjectItemDetails.xaml

```diff
@@ -440,16 +441,30 @@
 											 ContentTemplate="{StaticResource PageHeaderInformationTemplate}" />
 
 						<!-- RelatedWorkItems List -->
-						<ListView ItemsSource="{Binding ChildrenWorkItems.Result}"
-								  ItemTemplate="{StaticResource WorkItemTemplate}"
-								  Header="{Binding}"
-								  HeaderTemplate="{StaticResource PageHeaderInformationTemplate}"
-								  Footer="{Binding}"
-								  FooterTemplate="{StaticResource ListFooterTemplate}"
-								  SelectionMode="None"
-								  IsItemClickEnabled="True"
-								  ItemClick="ListView_ItemClick"
-								  Grid.Row="1" />
+						<android:GridView x:Name="GridViewRelatedWorkItemsList"
+							      ItemsSource="{Binding ChildrenWorkItems.Result}"
+						          ItemTemplate="{StaticResource WorkItemTemplate}"
+						          Header="{Binding}"
+						          HeaderTemplate="{StaticResource PageHeaderInformationTemplate}"
+						          Footer="{Binding}"
+						          FooterTemplate="{StaticResource ListFooterTemplate}"
+						          SelectionMode="None"
+						          IsItemClickEnabled="True"
+						          ItemClick="ListView_ItemClick"
+							      ItemContainerStyle="{StaticResource VerticalGridViewItemStyle}"
+						          Grid.Row="1"/>
+
```

Comment:
You should add a comment explaining why it's split between ListView and GridView